algo/src/main/kotlin/com/hartwig/actin/algo/evaluation/molecular/GeneHasActivatingMutation.kt

```diff
@@ -108,25 +99,16 @@ class GeneHasActivatingMutation(
 
     private fun evaluateVariant(variant: Variant, hasHighMutationalLoad: Boolean?): ActivationProfile {
         val isNoOncogene = variant.geneRole == GeneRole.TSG
-        val isGainOfFunction =
-            variant.proteinEffect == ProteinEffect.GAIN_OF_FUNCTION || variant.proteinEffect == ProteinEffect.GAIN_OF_FUNCTION_PREDICTED
         return if (variant.isReportable) {
             if (variant.driverLikelihood == DriverLikelihood.HIGH) {
                 return when {
                     isAssociatedWithDrugResistance(variant) -> profile(variant.event, ActivationWarningType.ASSOCIATED_WITH_RESISTANCE)
-                    !variant.isHotspot && !isGainOfFunction -> profile(
```

Comment:
Why would you want to remove this warning? 

It is still possible for a variant to be HIGH driver and not be a hotspot or have gain function.